src/utils/backoff_performer.cpp

```diff
@@ -27,6 +27,10 @@
  *
  */
 
+#ifdef _WIN32
+#define NOMINMAX
```

Comment:
This should probably be set as a definition at the cmake level if it's needed, rather than here in this particular file